doc/sphinx/source/recipes/recipe_ipccwg1ar6ch3.rst

```diff
@@ -198,6 +205,24 @@ User settings in recipe
    * y_min: set min of y-axis
    * y_max: set max of y-axis
 
+#. Script ipcc_ar6/zonal_westerly_winds.ncl
+
+   *Optional settings for variables*
+
+   * reference_dataset: reference dataset; REQUIRED when calculating
+     anomalies
+
+   *Optional settings for script*
+
```

Comment:
Could you check the indentation of this section and maybe remove the line breaks? Otherwise, the parameters are displayed with different formatting, see [here](https://esmvaltool--2984.org.readthedocs.build/en/2984/recipes/recipe_ipccwg1ar6ch3.html#user-settings-in-recipe).